hbshelpers/cssrtl.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Returns true if the given language is written from right to left (requires rtl css)
+ * 
+ * @param {string} locale 
+ * @returns {boolean}
+ */
+module.exports = function cssrtl(locale) {
```

Comment:
nit: what do you think of the name: isRTL?

Reply:
Are we able to put this function in static and refer to it from this helper? Ideally we have a list of RTL languages in one place so that we don't have to update them here and also in static/webpack-config.js

Reply:
that make sense. Updated!